### DISCUSSION TITLE
What is SSD M2 of this EFI

### DISCUSSION THREAD

**Discussion by XuanTruong-Github** [AndresGarciaSobrado91/MSI-Modern15-Hackintosh#27] [Q&A]
I'm using Samsung NVME 981 on my laptop .
Can i use this EFI for my laptop with my SSD Samsung  NVME 981 for hackintosh ?

**Comment by sysoev-dev** [comment]
I am using Samsung SSD 970 EVO Plus (nvme). Everything is great
